src/other/HeatMap.js

```diff
@@ -0,0 +1,73 @@
+"use strict";
+(function (root, factory) {
+    if (typeof define === "function" && define.amd) {
+        define(["d3", "../common/CanvasWidget", "simpleheat"], factory);
+    } else {
+        root.other_Heat = factory(root.d3, root.common_CanvasWidget, root.simpleheat);
+    }
+}(this, function (d3, CanvasWidget, simpleheat) {
+    function Heat() {
+        CanvasWidget.call(this);
+    }
+ 
+    Heat.prototype = Object.create(CanvasWidget.prototype);
```

Comment:
You will need to add a line defining the constructor - the unit tests will catch this once your up to date with master.